sql/moz-fx-data-shared-prod/amo_prod/desktop_addons_by_client_v1/query.sql

```diff
@@ -58,6 +43,65 @@ WITH per_client AS (
     submission_date,
     sample_id,
     client_id
+),
+per_clients_just_addons_base AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    client_id,
+    sample_id,
+    payload.info.addons
+  FROM
+    telemetry.main
+  WHERE
+    DATE(submission_timestamp) = @submission_date
+    AND client_id IS NOT NULL
```

Comment:
One nit: I'd move this `FROM` and `WHERE` clause to a new CTE that is queried by both `per_clients_just_addons_base` and `per_clients_without_addons`. That way we only define the filter/source once.